models/cam-fv/shell_scripts/cesm2_1/setup_pmo

```diff
@@ -0,0 +1,1277 @@
+#               A compset defined specifically for CAM assimilations is
+#               FHIST_DARTC6 = HIST_CAM60_CLM50%SP_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV
+#               For a list of the pre-defined component sets:
+#               > $CIMEROOT/scripts/create_newcase -list
+#               To create a variant compset, see the CESM documentation
+#               https://ncar.github.io/CAM/doc/build/html/users_guide/atmospheric-configurations.html
+#               and carefully incorporate any needed changes into this script.
+# resolution    Defines the horizontal resolution and dynamical core;
+#               see https://www.cesm.ucar.edu/models/cesm2/config/2.1.0/grids.html
+#               f19_f19_mg17  ... FV core at ~ 2 degree (19 means 1.9 degrees of latitude).
+#               f09_f09_mg17  ... FV core at ~ 1 degree (the 2nd f09 means CLM uses a .9 degree latitude grid)
+# cesmtag       The version of the CESM source code to use when building the model.
+#
+# Guidelines on what to change for a WACCM run are described in the 
+# models/cam-fv/readme.html documentation.
+# ==============================================================================
+
+setenv case            Test0
+setenv compset         HIST_CAM60_CLM50%BGC-CROP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV
+# setenv compset         FHIST_DARTC6
+setenv resolution      f09_f09_mg17
+setenv cesmtag         cesm2_1_relsd_m5.6
+# If cemstag >= cesm2_0_alpha10f and compset = FHIST_DARTC6 (or another non-supported):
+setenv compset_args   "--compset $compset"
+setenv compset_args   "${compset_args} --run-unsupported"
+
+# ==============================================================================
+# machines and directories:
+#
+# mach          Computer name
+# cesmdata      Location of some supporting CESM data files.
+# cesmroot      Location of the CESM code base.  This version of the script
+#               only supports version cesm2_0 or later, which is available from
+#               https://github.com/ESCOMP/cesm.
+# sourcemods    DART does not require a SourceMods directory in order to work with CESM.
+#               If you have modifications to CESM (including bug fixes), they should be provided 
+#               in a CESM-structured SourceMods directory, which this script expects to be in
+#               $user/$cesmtag/SourceMods.  Contact dart@ucar.edu for existing SourceMods.
+# caseroot      The CESM $CASEROOT directory, where the CESM+DART configuration files
+#               will be stored and the build commands will be executed.
+#               This should probably not be in scratch.
+#               This script will delete an existing caseroot with the same name,
+#               so this script and other things you want to preserve should be kept elsewhere.
+# dartroot      Location of the root of _your_ DART installation.
+# cime_output   $cime_root/$CASEROOT is the directory where CESM's build and run directories will be created.
```

Comment:
$cime_root/$CASEROOT does not match line 209
```
setenv cime_output /glade/scratch/${USER}
````

Reply:
Woops, that description is from an earlier version, which had the additional variable.
The comment should be
> # cime_output   $cime_output/$USER/$CASEROOT is the directory where CESM's build and run directories will be created.